packages/bitcoin/src/bitcoin.utils.ts

```diff
@@ -223,3 +224,8 @@ export function getTaprootAddress({ index, keychain, network }: GetTaprootAddres
 
   return payment.address;
 }
+
+export function mnemonicToRootNode(secretKey: string) {
+  const seed = mnemonicToSeedSync(secretKey);
+  return HDKey.fromMasterSeed(seed);
+}
```

Comment:
This is a kind of generic function that's needed for stacks too. Where else might it live?